src/plugin-import-Tiled.js

```diff
@@ -0,0 +1,280 @@
+/**
+🧱
```

Comment:
might be better to use the same emoji for all three since they're related? up to you though!